manual/mingw/tools/chocolateyUninstall.ps1

```diff
@@ -0,0 +1,3 @@
+. (Join-Path (Split-Path -parent $MyInvocation.MyCommand.Definition) chocolateyInclude.ps1)
+
+Uninstall-ChocolateyZipPackage "$packageName" "${packageName}Install.zip"
```

Comment:
It deletes the entire C:\tools\mingw64 (or mingw32) folder, if that's what you mean. It would be nice if it were a mirror of Install-ChocolateyZipPackage (where the `${packageName}Install.zip` construct is created).